Source/WebCore/rendering/OrderIterator.h

```diff
@@ -40,24 +40,40 @@ class RenderObject;
     
 class OrderIterator {
 public:
+    using iterator_category = std::forward_iterator_tag;
+    using value_type = RenderBox;
+    using difference_type = std::ptrdiff_t;
+    using pointer = RenderBox*;
+    using reference = RenderBox&;
+    pointer operator*();
+
+    OrderIterator& operator++();
+    // Post Increment
+    OrderIterator operator++(int);
+
+    bool operator==(const OrderIterator& other) const;
```

Comment:
It seems to be defined nowhere ?  We should remove this declaration

Reply:
The == operator is defined in OrderIterator.cpp.